app-text/pad/pad-1.0.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="A freeware notepad program, similar to Apple's Notepad app"
+DESCRIPTION="Pad is a freeware notepad program. Anyone who has used Apple's \
+Notepad app (included with the system software) should find it familiar.\
+ It supports multiple pages, the standard text editing commands \
+(Cut, Copy, Paste), as well as Drag 'n' Drop editing"
+HOMEPAGE="https://github.com/HaikuArchives/Pad"
+COPYRIGHT="1997 Jesse Hall"
+LICENSE="Pad"
+REVISION="1"
+SOURCE_URI="git://github.com/HaikuArchives/Pad"
+SOURCE_DIR=""
+CHECKSUM_SHA256=""
+#PATCHES=""
+ARCHITECTURES="x86 x86_gcc2 x86_64"
+PROVIDES="
+	pad = $portVersion
+	"
+REQUIRES="
+        haiku
+        "
+
+BUILD_REQUIRES="
+        "
+
+BUILD_PREREQUIRES="
+	haiku_devel >= $haikuVersion
+	cmd:gcc
+	cmd:make
+	makefile_engine
+"
+
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+mkdir $appsDir
+mv Pad $appsDir
```

Comment:
```diff
 INSTALL()
 {
-mkdir $appsDir
+	mkdir -p $appsDir
-mv Pad $appsDir
+	cp Pad $appsDir
-addAppDeskbarSymlink $appsDir/Pad
+	addAppDeskbarSymlink $appsDir/Pad
 }
```
* The `-p` in `mkdir -p $appsDir` is not really required but since it doesn't hurt we like to have it.
* We need a `cp` instead of a `mv` otherwise running haikuporter will fail the second time you invoke `hp pad`  because haikuporter will think the runtime is still there after the first run, but it won't if you use `mv`.